src/main/java/com/exasol/projectkeeper/validators/ReadmeValidator.java

```diff
@@ -0,0 +1,110 @@
+                    .message("The project's README.md does not reference the changelog.md file.'")
+                    .mitigation("Please add a link like '[Changelog](doc/changes/changelog.md)' to the changelog.md.")
+                    .toString()).build());
+        }
+        if (!content.contains(getBadges())) {
+            findings.add(ValidationFinding.withMessage(ExaError.messageBuilder("E-PK-62")
+                    .message("The project's README.md does not contain a valid badges block.")
+                    .mitigation("Please add or replace the following badges: \n{{badge}}.", getBadges()).toString())
+                    .build());
+        }
+        return findings;
+    }
+
+    @Override
+    protected String getTemplate() {
+        return "# " + this.projectName + NL + NL + //
+                getBadges() + NL + NL + //
+                "## Additional Information" + NL + NL + //
+                "* [Changelog](doc/changes/changelog.md)" + NL + //
+                "* [Dependencies](dependencies.md)";
+    }
+
+    public String getBadges() {
+        String badges = getBuildBadge() + NL;
+        if (this.enabledModules.contains(MAVEN_CENTRAL)) {
+            badges += getMavenCentralBadge() + NL;
+        }
+        badges += NL + //
+                sonarCloudBadge("Quality Gate Status", "alert_status") + NL + NL + //
+                sonarCloudBadge("Security Rating", "security_rating") + NL + //
+                sonarCloudBadge("Reliability Rating", "reliability_rating") + NL + //
+                sonarCloudBadge("Maintainability Rating", "sqale_rating") + NL + //
+                sonarCloudBadge("Technical Debt", "sqale_index") + NL + NL + //
+                sonarCloudBadge("Code Smells", "code_smells") + NL + //
+                sonarCloudBadge("Coverage", "coverage") + NL + //
+                sonarCloudBadge("Duplicated Lines (%)", "duplicated_lines_density") + NL + //
+                sonarCloudBadge("Lines of Code", "ncloc");
+        return badges;
+    }
+
+    private String sonarCloudBadge(final String name, final String metric) {
+        return getBadge(name, "https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3A" + this.artifactId
+                + "&metric=" + metric, "https://sonarcloud.io/dashboard?id=com.exasol%3A" + this.artifactId);
+    }
+
+    private String getBuildBadge() {
+        return getBadge("Build Status",
+                "https://github.com/exasol/" + this.repoName + "/actions/workflows/ci-build.yml/badge.svg",
+                "https://github.com/exasol/" + this.repoName + "/actions/workflows/ci-build.yml");
+    }
+
+    private String getMavenCentralBadge() {
+        return getBadge("Maven Central", "https://img.shields.io/maven-central/v/com.exasol/" + this.artifactId,
+                "https://search.maven.org/artifact/com.exasol/project-keeper-maven-plugin" + this.artifactId);
```

Comment:
This seems to be typo.
```suggestion
                "https://search.maven.org/artifact/com.exasol/" + this.artifactId);
```